### ISSUE TITLE
[machine learning] Which dataset to use?

### ISSUE THREAD

**Issue by CamiloInx** [KiwiCampusChallenge/Kiwi-Campus-Challenge#6]
Hi.

The dataset that you provided in this link: [http://benchmark.ini.rub.de/?section=gtsdb&subsection=dataset](url) for the challenge is a `benchmark` for traffic sign detection (GTSDB). As you pointed out in Task 6:

>  "Create an application that will use a chosen model and run inference on images saved in a particular directory. For each image in such directory you should show a window with the image together with its label." 

I understand that the object of the models is to perform classification in the traffic signs images, not detection. So I was wondering if with this benchmark I have to create a model that predicts only the label of the image and not the bounding box of the traffic sign in the image. If the before is correct, shouldn't we use the GTSRB (German Traffic Sign Recognition Benchmark) benchmark instead; which is a multi-class, single-image classification dataset and has more than 40 classes and more than 50,000 images in total.

In summary my question is: should we use the GTSDB or the GTSRB dataset?. If the dataset is GTSDB, would the task be to perform classification or classification with detection?, if it is classification with detection, could we use the GTSRB dataset to fine tune the classification task?

I provide the links for the two dataset pages:
GTSDB (Detection): [http://benchmark.ini.rub.de/?section=gtsdb&subsection=dataset](url)
GTSRB (Classification): [http://benchmark.ini.rub.de/?section=gtsrb&subsection=dataset](url) 

Thank you.

**Comment by KiwiCampusChallenge** [comment]
First of all, none of your links is working.

Now I clarify the instructions.
If you go to the link provided in the challenge, you will find a link to the full data set in the Download section; that compressed file contains images to perform object detection and object classification (the images for object classification are stored in folders corresponding to the label of the images), you should ignore (even in your download code) all images related to object detection.

Do not use different data from what it has been explained nor pretrained models
